hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/mutation/ExcludeSpanRuleUpdate.java

```diff
@@ -17,16 +18,19 @@ public interface ExcludeSpanRuleUpdate extends Identifiable {
 
   @GraphQLField
   @GraphQLName(NAME_KEY)
+  @GraphQLDescription("Update the rule name for the provided ruleId")
   @GraphQLNonNull
   String name();
 
   @GraphQLField
   @GraphQLName(SPAN_PROCESSING_FILTER_KEY)
+  @GraphQLDescription("Update the existing spanFilters for the input ruleId")
```

Comment:
Given this is a tree, you might say replace rather than update since the old value is discarded.